openquakeplatform/bin/deploy.sh

```diff
@@ -0,0 +1,303 @@
+#!/bin/bash                                                                     
+# Copyright (c) 2013-2019, GEM Foundation.
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake.  If not, see <http://www.gnu.org/licenses/>.
+
+# if [ $GEM_SET_DEBUG ]; then
+    set -x
+# fi
+set -e
+
+if [ -f /etc/openquake/platform/local_settings.py ]; then
+    GEM_IS_INSTALL=n
+else
+    GEM_IS_INSTALL=y
+fi
+
+sudo apt-get update
+sudo apt install -y git python-virtualenv wget
+
+GIT_REPO="oq-platform2"
+
+# if [ "$GEM_IS_INSTALL" == "y" ]; then
+    # delete all folder used
+    sudo rm -rf env $GIT_REPO geonode oq-platform-taxtweb oq-platform-building-class oq-platform-ipt oq-platform-data
+    
+    # if exists, delete postgres:
+    sudo apt-get --purge remove -y postgresql postgresql-9.5 postgresql-9.5-postgis-2.2 postgresql-9.5-postgis-scripts postgresql-client-9.5 postgresql-client-common postgresql-common postgresql-contrib-9.5
+# fi
+# Create and source virtual env
+virtualenv env
+source $HOME/env/bin/activate
+
+sudo apt install -y python-dev libpq-dev libgdal-dev openjdk-8-jdk-headless
+
+sudo apt-get install -y postgresql-9.5-postgis-2.2 postgresql-9.5-postgis-scripts curl xmlstarlet supervisor
+pip install numpy
+sudo apt install -y apache2 tomcat7
+python -m pip install "django<2"
+pip install django-nested-inline
+pip install django_extras
+pip install -e git+git://github.com/gem/django-chained-selectbox.git@pla26#egg=django-chained-selectbox-0.2.2
+pip install -e git+git://github.com/gem/django-nested-inlines.git@pla26#egg=django-nested-inlines-0.1.4
+pip install -e git+git://github.com/gem/django-chained-multi-checkboxes.git@pla26#egg=django-chained-multi-checkboxes-0.4.1
+pip install -e git+git://github.com/gem/wadofstuff-django-serializers.git@pla26#egg=wadofstuff-django-serializers-1.1.2
+pip install scipy
```

Comment:
`scipy` breaks easily, I would pin it to a specific release, or even better I would put everything in a `requirements.txt` to better manage requirements.